test/e2e/app-dir/actions/app/revalidate-2/page.js

```diff
@@ -33,9 +34,7 @@ export default async function Page() {
       </form>
       <p>
         random cookie:{' '}
-        <span id="random-cookie">
-          {JSON.stringify((await cookies()).get('random'))}
-        </span>
+        <span id="random-cookie">{JSON.stringify(randomCookie)}</span>
```

Comment:
This change was needed because when PPR was enabled, the components fetch data was being added to the resume data cache (or RDC) which was causing static/dynamic sheering.